kiva/fonttools/font_manager.py

```diff
@@ -673,7 +674,7 @@ def createFontList(fontfiles, fontext='ttf'):
     a list of TrueType fonts.  An AFM font list can optionally be
     created.
     """
-
+    # FIXME: This function is particularly difficult to debug
```

Comment:
I guess I should really highlight that this particular function caused me a lot of headaches. I think there are several issues in enable that stems from the error gobbling that this function does. #259 comes to mind, there are probably more.